core/trie2/triedb/pathdb/layertree_test.go

```diff
@@ -142,41 +149,41 @@ func (m *mockTrieNode) IsLeaf() bool    { return len(m.blob) == 0 }
 // layerTracker tracks trie nodes across different layers to simplify testing
 type layerTracker struct {
 	// Mimic the layer tree by mapping the state root to the nodes
-	classNodes           map[felt.Felt]map[trieutils.Path]trienode.TrieNode
-	contractNodes        map[felt.Felt]map[trieutils.Path]trienode.TrieNode
-	contractStorageNodes map[felt.Felt]map[felt.Address]map[trieutils.Path]trienode.TrieNode
+	classNodes           map[felt.Hash]map[trieutils.Path]trienode.TrieNode
+	contractNodes        map[felt.Hash]map[trieutils.Path]trienode.TrieNode
+	contractStorageNodes map[felt.Hash]map[felt.Address]map[trieutils.Path]trienode.TrieNode
 
 	// Tracks all unique and latest nodes in the layer tree
 	classPaths           map[trieutils.Path]trienode.TrieNode
 	contractPaths        map[trieutils.Path]trienode.TrieNode
 	contractStoragePaths map[felt.Address]map[trieutils.Path]trienode.TrieNode
```

Comment:
Is this `Address` referencing `StorageAddress`, if that's the case, then it is being miss-used. 

`Address` strictly refers to a Starknet contract address – this has been discussed both in dailies and slack.